staging/src/k8s.io/apimachinery/pkg/api/validation/objectmeta.go

```diff
@@ -199,10 +199,6 @@ func ValidateInitializers(initializers *metav1.Initializers, fldPath *field.Path
 			allErrs = append(allErrs, field.Invalid(fldPath.Child("pending").Index(i), initializer.Name, msg))
 		}
 	}
-	allErrs = append(allErrs, validateInitializersResult(initializers.Result, fldPath.Child("result"))...)
```

Comment:
Restored. I'll add a unit test in https://github.com/kubernetes/kubernetes/pull/51283. The validation code has other problems.